<a href="https://colab.research.google.com/github/mklatzke/IndividualMovieAnalysisProject/blob/main/Movie_Analysis_Klatzke.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


In [2]:
import pandas as pd
file_path = '/content/drive/MyDrive/movies_dataset.csv' # Paste your path here using the Files tab on the left hand side
movies = pd.read_csv(file_path)

# 1. Are movies released in odd-numbered years more profitable than movies from even-numbered years?

In [12]:
from scipy import stats
# categorize release dates by odd or even year
movies['Date'] = pd.to_datetime(movies['ReleaseDate'], dayfirst=True) # convert String values to DateTimes
movies['YearsOddEven'] = movies['Date'].dt.year % 2 # new column where even years = 0 and odd = 1
movies['YearsOddEven'] = movies['YearsOddEven'].map({0: 'Even', 1: 'Odd'}) # change labels to Even or Odd
even_revenue = movies[movies['YearsOddEven'] == 'Even']['Global_BoxOfficeUSD']
odd_revenue = movies[movies['YearsOddEven'] == 'Odd']['Global_BoxOfficeUSD']

# run two-sample t-test
t_stat, p_value = stats.ttest_ind(even_revenue, odd_revenue)
print(p_value)

0.4710319611283329


The null hypothesis in a two-sample t-test is that there is no difference between the means of the groups. This test compares the mean global box offices between movies released in odd and even years. The p-value is 0.4710319611283329, so the test fails to reject the null hypothesis. There is no statistical difference between the average revenue of movies released in odd-numbered years and even-numbered years.

# 2. How much of the total box office gross comes from opening day ticket sales?